### PR TITLE
[Ubuntu] Add PGP validation for swift

### PR DESCRIPTION
**Pull request by sergei-pyshnoi**
# Description

This PR adds PGP validation for swift installation

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/swift.sh [hunk 1]
```diff
@@ -15,6 +15,17 @@ swift_tar_name="swift-$swift_version-RELEASE-ubuntu$image_label.tar.gz"
 swift_tar_url="https://swift.org/builds/swift-$swift_version-release/ubuntu${image_label//./}/swift-$swift_version-RELEASE/$swift_tar_name"
 download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 
+# Verifing pgp signature using official swift pgp key. Referring to https://www.swift.org/install/linux/#Installation-via-Tarball
+# Download swift pgp key
+download_with_retries "https://swift.org/keys/all-keys.asc" "/tmp" "all-keys.asc"
+# Import swift pgp key
+gpg --import /tmp/all-keys.asc
```
Comment (vpolikarpov-akvelon) [inline images/linux/scripts/installers/swift.sh:22]: Are we good with keeping this public key in keyring? If not then we should either remove it afterwards or import it to temporary keyring instead (using `--no-default-keyring --keyring <keyring-file-name>`)

Comment (sergei-pyshnoi) [reply images/linux/scripts/installers/swift.sh:22]: I didn`t see nothing bad in keeping keys. @mikhailkoliada What do you think?

Comment (mikhailkoliada) [reply images/linux/scripts/installers/swift.sh:22]: yes, lets clean it up after we use it

Comment (sergei-pyshnoi) [reply images/linux/scripts/installers/swift.sh]: fixed

File: images/linux/scripts/installers/swift.sh [hunk 2]
```diff
@@ -19,11 +19,13 @@ download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 # Download swift pgp key
 download_with_retries "https://swift.org/keys/all-keys.asc" "/tmp" "all-keys.asc"
 # Import swift pgp key
-gpg --import /tmp/all-keys.asc
+gpg --no-default-keyring --keyring swift --verify --import /tmp/all-keys.asc
```
Comment (vpolikarpov-akvelon) [inline images/linux/scripts/installers/swift.sh:22]: `--verify` and  `--import` options can't be used together as both represent actions

Comment (sergei-pyshnoi) [reply images/linux/scripts/installers/swift.sh]: oops... Fixed

**Review by vpolikarpov-akvelon — APPROVED** [review]

File: images/linux/scripts/installers/swift.sh [hunk 3]
```diff
@@ -15,6 +15,19 @@ swift_tar_name="swift-$swift_version-RELEASE-ubuntu$image_label.tar.gz"
 swift_tar_url="https://swift.org/builds/swift-$swift_version-release/ubuntu${image_label//./}/swift-$swift_version-RELEASE/$swift_tar_name"
 download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 
+# Verifing pgp signature using official swift pgp key. Referring to https://www.swift.org/install/linux/#Installation-via-Tarball
```
Comment (samvantran) [inline images/linux/scripts/installers/swift.sh:18]: typo should be `gpg`

Comment (sergei-pyshnoi) [reply images/linux/scripts/installers/swift.sh:18]: @samvantran I thought "gpg" it is OpenPGP encryption and signing tool. Also here is part of public key:
```
-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: 14F2 6682 D091 6CDD 81E3  7B6D 61B7 B526 D98F 0353

xsFNBFWpQAQBEAC+9wVlwGLy8ILCybLesuB3KkHHK+Yt1F1PJaI30X448ttGzxCz
PQ......
```

Comment (samvantran) [reply images/linux/scripts/installers/swift.sh:18]: yeah I realized my mistake when I went to read the swift docs (thanks for including the link in the comment)

**Review by Alexey-Ayupov — APPROVED** [review]

**Pull request merged**